snapcraft/plugins/waf.py

```diff
@@ -65,6 +65,18 @@ def get_build_properties(cls):
         # change in the YAML Snapcraft will consider the build step dirty.
         return ['configflags']
 
+    def env(self, root):
+        env = super().env(root)
+        if self.project.is_cross_compiling:
+            env.extend([
+                'CC={}-gcc'.format(self.project.arch_triplet),
+                'CXX={}-g++'.format(self.project.arch_triplet),
+            ])
+        return env
+
+    def enable_cross_compilation(self):
```

Comment:
Something still missing - if so drop this and line 160.
Otherwise I think a comment why it is there for people like me would be very kind.